rmgpy/pdep/me.pyx

```diff
@@ -39,21 +39,21 @@ import rmgpy.constants as constants
 ################################################################################
 
 
-cpdef generate_full_me_matrix(network, bint products=True):
+cpdef generate_full_me_matrix(network, bint products=True, bint exclude_association=False, bint neglect_high_energy_collisions=False, high_energy_rate_tol=0.01):
```

Comment:
Does high_energy_rate_tol not need a typing in front of it?

Reply:
fixed